src/libraries/Common/src/System/Experimentals.cs

```diff
@@ -28,8 +28,7 @@ internal static class Experimentals
         // X86Base.DivRem is experimental since performance is not as optimized as T.DivRem
         internal const string X86BaseDivRemDiagId = "SYSLIB5004";
 
-        // System.Formats.Nrbf is experimental
-        internal const string NrbfDecoderDiagId = "SYSLIB5005";
+        // System.Formats.Nrbf was experimental, do not reuse "SYSLIB5005"
```

Comment:
Thank you for leaving this comment here! We need to keep doing that each time something is removed from here.